hack/testing-olm/test-200-verify-es-metrics-access.sh

```diff
@@ -59,78 +62,9 @@ if [ "${DO_SETUP:-true}" == "true" ] ; then
-  oc create clusterrolebinding --clusterrole=basic-user view-${CLUSTERROLE} --serviceaccount=${TEST_NAMESPACE}:${AUTHORIZED_SA}
-fi
-
-os::log::info "---------------------------------------------------------------"
-os::log::info "Creating RBAC for unauthorised serviceaccount to verify metrics"
-os::log::info "---------------------------------------------------------------"
-result=$(oc get clusterrolebinding view-${CLUSTERROLE}-unauth --ignore-not-found ||:)
-if [ "$result" == "" ] ; then
-  os::log::info Binding ${UNAUTHORIZED_SA} to be cable of getting namespaces
-  oc create clusterrolebinding --clusterrole=basic-user view-${CLUSTERROLE}-unauth --serviceaccount=${TEST_NAMESPACE}:${UNAUTHORIZED_SA}
-fi
-
-es_pod=$(oc -n ${TEST_NAMESPACE} get pod -l component=elasticsearch -o jsonpath={.items[0].metadata.name})
-
-os::log::info "---------------------------------------------------------------"
-os::log::info "Waiting until elasticsearch cluster initialization is completed"
-os::log::info "---------------------------------------------------------------"
-os::cmd::expect_success "oc -n ${TEST_NAMESPACE} wait --for=condition=Ready pod/${es_pod} --timeout=120s"
-
-push_test_script_to_es(){
-  es_pod=$1
-  token=$2
-  service_ip=elasticsearch-metrics.${TEST_NAMESPACE}.svc
-  echo "curl -ks -o /tmp/metrics.txt https://${service_ip}:60001/_prometheus/metrics -H Authorization:'Bearer ${token}' -w '%{response_code}\n'" > /tmp/test
-  os::cmd::expect_success "oc -n ${TEST_NAMESPACE} cp /tmp/test ${es_pod}:/tmp/test -c elasticsearch"
-  os::cmd::expect_success "oc -n ${TEST_NAMESPACE} exec ${es_pod} -c elasticsearch -- chmod 777 /tmp/test"
-}
-
-os::log::info "---------------------------------------------------------------------------"
-os::log::info "Checking ${AUTHORIZED_SA} ability to read metrics through metrics service"
-os::log::info "---------------------------------------------------------------------------"
-token=$(oc -n ${TEST_NAMESPACE} serviceaccounts get-token $AUTHORIZED_SA)
-push_test_script_to_es $es_pod $token
-os::cmd::expect_success_and_text "oc -n ${TEST_NAMESPACE} exec ${es_pod} -c elasticsearch -- bash -c /tmp/test" '200'
 
-os::log::info "---------------------------------------------------------------------------"
-os::log::info "Checking ${UNAUTHORIZED_SA} ability to read metrics through metrics service"
-os::log::info "---------------------------------------------------------------------------"
-token=$(oc -n ${TEST_NAMESPACE} serviceaccounts get-token $UNAUTHORIZED_SA)
-push_test_script_to_es $es_pod $token
-os::cmd::expect_success_and_text "oc -n ${TEST_NAMESPACE} exec ${es_pod} -c elasticsearch -- bash -c /tmp/test" '403'
+CLUSTERROLE=${CLUSTERROLE} AUTHORIZED_SA=${AUTHORIZED_SA} UNAUTHORIZED_SA=${UNAUTHORIZED_SA} \
+  TEST_OPERATOR_NAMESPACE=${TEST_NAMESPACE} \
+  TEST_WATCH_NAMESPACE=${TEST_NAMESPACE} \
+  go test -v ./test/e2e-olm/... -kubeconfig=${KUBECONFIG} -parallel=1 -timeout 1500s -run TestElasticsearchOperatorMetrics
```

Comment:
Missing passing this test to the junit report like [here](https://github.com/openshift/elasticsearch-operator/pull/643/files#diff-fa5e2f28a843bccb27a13c4417f9866a2cc257327be992150e11a92408ff0e59R56)

Reply:
Fixed. Thanks!